documentation/src/docs/asciidoc/release-notes/release-notes-5.11.0-RC1.adoc

```diff
@@ -60,7 +60,8 @@ repository on GitHub.
   `assertInstanceOf` methods introduced in `5.8` version.
 * New generators in `DynamicTest` that take a `Stream`/`Iterator` of `Named<Executable>`
   along with a convenient `NamedExecutable` interface that can simplify writing dynamic
-  tests, in particular in recent version of Java that support records.
+  tests, in particular in recent versions of Java that support records.
```

Comment:
Thanks for fixing my typo! :+1:

Reply:
Oh, thanks for catching the release notes gap!